src/modules/myConversations.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Utilities for showing agent level interaction and evaluation details
+ *
+ * @module modules/myConversations
+ *
+ * @since 1.3.0
+ */
+import BaseApi from './base';
+
+/**
+ * Utilities for showing agent level interaction and evaluation details
+ *
+ * @extends module:modules/base~BaseApi
+ *
+ * @since 1.3.0
+ */
+
+class MyConversationsApi extends BaseApi {
+    /**
+     * Show an interaction by ID.
+     *
+     * Required Permissions:
+     * * ALL Of
+     *     * quality:evaluation:view
```

Comment:
This permission is no longer needed on this call.  I've got this cleaned up on my branch so I'll just push this shortly.